packages/react-query/src/NodeVersion.tsx

```diff
@@ -14,13 +14,15 @@ interface Props extends BareProps {
 
 export default function NodeVersion ({ children, className, label, style }: Props): React.ReactElement<Props> {
   const { systemVersion } = useApi();
+  // change version 1.2.0-rc1-4827374-x86_64-linux-gnu to 1.2.0
+  const nodeVer = systemVersion.split('-')
 
   return (
     <div
       className={className}
       style={style}
     >
-      {label || ''}{systemVersion}{children}
+      {label || ''}{nodeVer[0]}{children}
```

Comment:
is it possible to style this instead e.g. `text-overflow: ellipsis;`
the version string could be more complicated and the split will happen in differnt places

Reply:
I tried this css property -` text-overflow: ellipsis;`
However it gets struck-through (crossed out) cause parent uses **@media queries** in css at https://github.com/cennznet/ui/blob/9f1f7d6e2b89ca36bfbe400463c82ac8eafea159/packages/apps/src/SideBar/SideBarScroll.tsx#L7

https://stackoverflow.com/questions/3047056/what-do-the-crossed-style-properties-in-google-chrome-devtools-mean